HUD/hud_scraper.py

```diff
@@ -0,0 +1,177 @@
+import os
+import sys
+import json
+import pandas as pd
+from pymongo import MongoClient, TEXT, errors
+import random
+from tqdm import tqdm
+import re
+from collections import OrderedDict
+import requests
+from datetime import datetime, date
+
+if __package__:  # if script is being run as a module
+    from ..shelterapputils.utils import check_similarity, refresh_ngrams
+else:  # if script is being run as a file
+    _i = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+    if _i not in sys.path:
+        # add parent directory to sys.path so utils module is accessible
+        sys.path.insert(0, _i)
+    del _i  # clean up global name space
+    from shelterapputils.utils import (
+        check_similarity, refresh_ngrams,
+        make_ngrams, locate_potential_duplicate,
+        distance, insert_services
+    )
+# Establish global variables
+client = MongoClient(
```

Comment:
this can also be moved into utils.py? since most scrapers will use the same client?